awx/main/analytics/core.py

```diff
@@ -146,30 +144,40 @@ def gather(dest=None, module=None, collection_type='scheduled'):
 
 def ship(path):
     """
-    Ship gathered metrics via the Insights agent
+    Ship gathered metrics to the Insights API
     """
+    if not path:
+        logger.error('Automation Analytics TAR not found')
+        return
+    if "Error:" in str(path):
+        return
     try:
-        agent = 'insights-client'
-        if shutil.which(agent) is None:
-            logger.error('could not find {} on PATH'.format(agent))
-            return
         logger.debug('shipping analytics file: {}'.format(path))
-        try:
-            cmd = [
-                agent, '--payload', path, '--content-type', settings.INSIGHTS_AGENT_MIME
-            ]
-            output = smart_str(subprocess.check_output(cmd, timeout=60 * 5))
-            logger.debug(output)
-            # reset the `last_run` when data is shipped
-            run_now = now()
-            state = TowerAnalyticsState.get_solo()
-            state.last_run = run_now
-            state.save()
-
-        except subprocess.CalledProcessError:
-            logger.exception('{} failure:'.format(cmd))
-        except subprocess.TimeoutExpired:
-            logger.exception('{} timeout:'.format(cmd))
+        base_url = getattr(settings, 'AUTOMATION_ANALYTICS_URL', None)
+        if not base_url:
+            logger.error('AUTOMATION_ANALYTICS_URL is not set')
+            return
+        url = base_url + '/api/ingress/v1/upload'
```

Comment:
Should this part of the url be hardcoded?

Reply:
@benthomasson can you think of a reason we wouldn't want to include this in the setting?  After all, if the base of the url may be liable to change, I think it is fair to say the rest of the url could as well.

Reply:
Allowing this to be user changeable might insulate us from API path changes.  It will also allow users to break this functionality without knowing that they broke it.

Reply:
@one-t I talked to @wenottingham and @benthomasson and they agree we could hardcode this.  I have updated the PR.